webpack.config.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
 
   resolve: {
     extensions: [".ts", ".tsx", ".js", ".jsx"],
+    alias: {src: path.resolve(__dirname, "src")},
```

Comment:
do you prefer using the `src/` prefix for paths in general? I was omitting because I wanted to force people to use shallow directory structure

Reply:
The main reason for this is using `src/index` as the only entry point (for exported components even inside a package) is that we don't even need to know if the project structure is shallow of somehow structured.
I believe that sooner or later when more components are extracted - we probably need to adjust the project structure and this guarantees that no needs to modify import paths in every component.
And it shouldn't force us to create a nested structure.